Style formulator 9000 page

BrayonCrayon opened the pull request.

heyitskei commented on src/App.vue:
```diff
@@ -7,10 +7,14 @@ import AtomeqNav from '@/components/AtomeqNav.vue';
   <AtomeqLayout>
     <template #nav>
-      <AtomeqNav :isAuthed="true" />
+      <div class="">
```
does this need an empty class declaration? 🤔

BrayonCrayon replied:
Removed empty class, nice catch!

heyitskei commented on src/components/AtomeqNav.vue:
```diff
@@ -14,63 +14,59 @@ const showMenu = () => {
-      <p v-if="isAuthed" class="font-semibold">Logout</p>
+      <div v-if="isAuthed" class="flex gap-x-4 flex-grow justify-center">
+        <RouterLink :to="{ name: 'table' }" class="font-semibold cursor-pointer">Table</RouterLink>
+        <RouterLink :to="{ name: 'formulator' }" class="font-semibold cursor-pointer"
```
very nitpicky nitpick, but the way it's closing on the new line is killing me 😢

BrayonCrayon replied:
Good catch, I'll correct it

heyitskei reviewed (dismissed):
Non-blocking comments, approved! Looking ✨ beautiful ✨

heyitskei commented:
Linter is failing, otherwise lgtm! ❤️

BrayonCrayon committed:
Style Formulator 9000 Page
- Formulator page to have inputs for users equation
- Make the page mobile friendly
- Style tweaks to make the app look and feel consistent through mobile and desktop views

Co-authored-by: Nick Byam <n.byam@vehikl.com>
Co-authored-by: Ekaterina Stroganova <e.stroganova@vehikl.com>
Co-authored-by: Justin Bates <j.bates@vehikl.com>

heyitskei reviewed (dismissed):
:shipit:

heyitskei reviewed (dismissed):
🚢

heyitskei commented:
Typecheck is failing the pipeline due to AI shenanigans, will need a closer look before it can go in 😢

heyitskei reviewed (approved):
Just ship her already please
